types/tailwindcss/tailwindcss-tests.ts

```diff
@@ -0,0 +1,2500 @@
+import resolveConfig from 'tailwindcss/resolveConfig';
```

Comment:
```suggestion
import resolveConfig  = require('tailwindcss/resolveConfig');
```

Reply:
Why like this?

This is different from the official documentation

```ts
import resolveConfig from 'tailwindcss/resolveConfig'
```

I do not think this is a good change

Reply:
The suggestion was just to use a TS-style import, which is good advice in general.  If the documentation shows that the library should be imported this way, then it's probably better to test it this way.